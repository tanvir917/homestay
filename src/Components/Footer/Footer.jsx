import React from "react";

export default function Footer() {
  const year = new Date().getFullYear();

  return (
    <footer className="footer">
      <h2>Homestay MIT</h2>
      <p>
        &copy; <span>{year}</span> Homestaymit.com - All Rights Copyright
        Reserved To{" "}
        <a
          target="_blank"
          rel="noreferrer"
          href="https://www.github.com/tanvir917"
        >
          SD Lab Group Projet: Tanvir, Rohit, Devansh
        </a>
      </p>
    </footer>
  );
}
