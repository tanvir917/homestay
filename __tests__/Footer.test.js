import React from 'react'
import TestRenderer from 'react-test-renderer';
import Footer from '../src/Components/Footer/Footer';


it('renders correctly', () => {
  const tree = TestRenderer
    .create(<Footer />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});