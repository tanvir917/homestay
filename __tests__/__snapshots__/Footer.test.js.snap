// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="footer"
>
  <h2>
    Homestay MIT
  </h2>
  <p>
    Â© 
    <span>
      2023
    </span>
     Homestaymit.com - All Rights Copyright Reserved To
     
    <a
      href="https://www.github.com/tanvir917"
      rel="noreferrer"
      target="_blank"
    >
      SD Lab Group Projet: Tanvir, Rohit, Devansh
    </a>
  </p>
</footer>
`;
